projects/wave-core-new/src/lib/project/notifications/notifications.component.ts

```diff
@@ -0,0 +1,31 @@
+import {Component, OnInit, ChangeDetectionStrategy} from '@angular/core';
+import {NotificationService} from '../../notification.service';
+import {Notification} from '../../notification.service';
+
+@Component({
+    selector: 'wave-notifications',
+    templateUrl: './notifications.component.html',
+    styleUrls: ['./notifications.component.scss'],
+    changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class NotificationsComponent implements OnInit {
+    notifications: Array<Notification>;
+
+    constructor(private notificationService: NotificationService) {
+        this.notifications = [];
+    }
+
+    ngOnInit(): void {
+        this.notifications = this.notificationService.notifications;
+        this.notifications = this.notifications.reverse();
+    }
+
+    removeAllNotifications(): void {
+        this.notifications = [];
```

Comment:
this only clears the current view of the notifications. If I close and open the settings again the notifications reappear

Reply:
d27ffd9b23a6bb4f382ea9cea14eb83f3fb9ba10